### PR TITLE
ChatFeed notebook improvements

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Improvements based on review of the `ChatFeed` notebook.

Overall the notebook is great. I've made some changes to make it easier to see what goes on when I run the notebook interactively.

- Put `.send` in its own cell
- Add `await sleep` in some places

To make the Feed more fun and engaging

- I've changed the user emoji from 🧑 to 👤. The 🧑 represents a genderless person. The color and not least smile helps put your into a positive mood.
- I've changed the `callback` emoji from 🤖 to the below avatar. 

I think this avatar better signals positivity and fun. If you agree we should get the `.png` rescaled and uploaded to the holoviz cdn for optimal performance.

## Assets

I've made some gifs to also display the OPENAI examples

openai-acreate

bot-conversation

These should be uploaded to our CDN.

### CODE REVIEW

File: examples/reference/widgets/ChatFeed.ipynb [hunk 1]
```diff
@@ -258,14 +274,48 @@
...
+    "chat_feed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: IT'S LIKE THE its not sleeping but just displaying right away. The spinner is not displayed either. Andrew should look at this. `async` functions with `return` should also work.\n",
```
Comment (MarcSkovMadsen) [inline examples/reference/widgets/ChatFeed.ipynb:309]: It like this example with an `async` function with a `return` statement does not work correctly. Its like the `await sleep` is never run. And the `LoadingIndicator` is not triggered either.

Would you take a look @ahuang11 ? Thanks.

**Comment by ahuang11** [comment]
>I've changed the user emoji from 🧑 to 👤.
I think you mean from 👤 to 🧑*; I actually prefer the shadowy figure since a similar image is used as the default in most social media, but I don't really have strong feelings for this.

In regards to the callback avatar, I prefer not to use a URL for an avatar since it requires the user to have an internet connection. Also, it's using a Ubuntu(?) mascot(?) which I don't think should be associated with Panel.

**Comment by ahuang11** [comment]
I will take a closer look tonight and hopefully merge so there's not so many diverging PRs at once.

File: examples/reference/widgets/ChatFeed.ipynb (hunk 1 above)
Comment (ahuang11) [reply examples/reference/widgets/ChatFeed.ipynb:309]: I'm a bit confused with this request. Since `respond=False`, the callback will not execute.

**Review by ahuang11 — APPROVED** [review]

**Comment by ahuang11** [comment]
I appreciate your comments and thanks for adding the GIFs; I made tweaks based on your suggestions and merging now.

**Pull request merged**